code/network/lib/Loot/Network/ZMQ/Server.hs

```diff
@@ -173,7 +171,11 @@ runBroker = do
             Publish k v -> publish k v
 
     let frontToListener = \case
-            (cId:"":msgT:msg) -> do
+            [cId,"","getid"] -> do
```

Comment:
```suggestion
            [cId, "", "getid"] -> do
```
We aren't running low on screen width and neither we're going to print the code on narrower analogs of paper towels. Please, do not forget spaces.

Reply:
> Please, do not forget spaces.

I don't put them intentionally because I find it easier to read.

>We aren't running low on screen width

I am actually limited to 82 symbols width.